python/langsmith/client.py

```diff
@@ -1165,6 +1165,50 @@ def read_project(
             **response.json(), _host_url=self._host_url
         )
 
+    def get_test_results(
+        self,
+        project_id: Optional[ID_TYPE] = None,
```

Comment:
```suggestion
        *,
        project_id: Optional[ID_TYPE] = None,
```

Reply:
not sure if you like, but i like forcing named invocation for whatever could be a literal since it makes user code more readable (but longer to type), but I like typing more code

Reply:
It's more consistent with the rest of the API so ya good point. I've grown tired of typing "project_id" so often

Reply:
but it's an opportunity to not think about anything since and let muscle memory engage for a few seconds